cg/services/orders/submitters/fastq_order_submitter.py

```diff
@@ -1,24 +1,23 @@
-from cg.models.orders.order import OrderIn
+from cg.services.order_validation_service.workflows.fastq.models.order import FastqOrder
+from cg.services.order_validation_service.workflows.fastq.validation_service import (
+    FastqValidationService,
+)
 from cg.services.orders.store_order_services.store_fastq_order_service import StoreFastqOrderService
 from cg.services.orders.submitters.order_submitter import OrderSubmitter
-from cg.services.orders.validate_order_services.validate_fastq_order import (
-    ValidateFastqOrderService,
-)
 
 
 class FastqOrderSubmitter(OrderSubmitter):
     """Submitter for fastq orders."""
 
     def __init__(
         self,
-        order_validation_service: ValidateFastqOrderService,
+        order_validation_service: FastqValidationService,
```

Comment:
The whole thing is built a bit backwards now since the order_validation_service is never used within this class

Reply:
then when is the validation done?